ddp-workspace/projects/ddp-mbc/src/app/components/about-us/about-us.component.ts

```diff
@@ -68,126 +68,125 @@ import { ToolkitConfigurationService } from 'toolkit';
-                        <a href="https://shaysharpespinkwishes.org/" target="_blank"><img lazy-resource class="partner-logo" src="./assets/images/logo-SSLogo-noBG.PNG" alt="Sarcoma Alliance Logo"></a>
+                        <a href="https://shaysharpespinkwishes.org/" target="_blank"><img lazy-resource class="partner-logo" src="./assets/images/logo-SSLogo-noBG.PNG" [alt]="'Toolkit.Common.LogoAlts.Shay' | translate"></a>
                     </div>
                     <div class="col-lg-3 col-md-3 col-sm-3 col-xs-12 Message-partners">
-                        <a href="https://www.metastasis-research.org/" target="_blank"><img lazy-resource class="partner-logo" src="./assets/images/logo-MRS_LogoColor.jpg" alt="Sarcoma Alliance Logo"></a>
+                        <a href="https://www.metastasis-research.org/" target="_blank"><img lazy-resource class="partner-logo" src="./assets/images/logo-MRS_LogoColor.jpg" [alt]="'Toolkit.Common.LogoAlts.MRS' | translate"></a>
                     </div>
                     <div class="col-lg-3 col-md-3 col-sm-3 col-xs-12 Message-partners">
-                        <a href="https://sistersrus.org/" target="_blank"><img lazy-resource class="partner-logo" src="./assets/images/logo-highresolutionpng.png" alt="Sarcoma Alliance Logo"></a>
+                        <a href="https://sistersrus.org/" target="_blank"><img lazy-resource class="partner-logo" src="./assets/images/logo-highresolutionpng.png" [alt]="'Toolkit.Common.LogoAlts.Sisters' | translate"></a>
                     </div>
                 </div>
                 <div class="row">
                     <div class="col-lg-4 col-md-4 col-sm-4 col-xs-12 Message-partners">
-                        <img lazy-resource class="partner-logo" src="./assets/images/logo-hmn.png" alt="SARC Logo">
+                        <img lazy-resource class="partner-logo" src="./assets/images/logo-hmn.png" [alt]="'Toolkit.Common.LogoAlts.HMN' | translate">
                     </div>
                     <div class="col-lg-4 col-md-4 col-sm-4 col-xs-12 Message-partners">
-                        <img lazy-resource class="partner-logo" src="./assets/images/logo-min_coalition.png" alt="Sarcoma Alliance Logo">
+                        <img lazy-resource class="partner-logo" src="./assets/images/logo-min_coalition.png" [alt]="'Toolkit.Common.LogoAlts.Min' | translate">
                     </div>
                     <div class="col-lg-4 col-md-4 col-sm-4 col-xs-12 Message-partners">
-                        <a href="https://mbccanada.org/" target="_blank"><img lazy-resource class="partner-logo" src="./assets/images/logo_mbccanada.png" alt="Sarcoma Alliance Logo"></a>
+                        <a href="https://mbccanada.org/" target="_blank"><img lazy-resource class="partner-logo" src="./assets/images/logo_mbccanada.png" [alt]="'Toolkit.Common.LogoAlts.MBCCanada' | translate"></a>
                     </div>
                 </div>
             </section>
```

Comment:
Found one that looks incorrect:
<img width="872" alt="Screen Shot 2020-04-16 at 4 07 30 PM" src="https://user-images.githubusercontent.com/29984380/79501773-99f9ea00-7ffc-11ea-8d35-b90a863e7844.png">